crates/bevy_utils/src/short_names.rs

```diff
@@ -1,62 +1,98 @@
-use alloc::string::String;
-
 /// Shortens a type name to remove all module paths.
```

Comment:
Nit: I’d suggest also mentioning the lazy evaluation here as well for better visibility (i.e. "Lazily shortens a type name...")